Makefile

```diff
@@ -65,12 +65,11 @@ coverage: ## check code coverage quickly with the default Python
 		$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/bigchaindb.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ bigchaindb
```

Comment:
I think that building `docs/bigchaindb.rst` was part of the BigchainDB release process, see https://github.com/bigchaindb/bigchaindb/blob/master/Release_Process.md

It's not there anymore, as far as I can see. Should something be _added_ to the release process?